nnvm/python/nnvm/frontend/tensorflow.py

```diff
@@ -975,6 +984,26 @@ def from_tensorflow(self, graph):
 
                 attr = self._parse_attr(node.attr)
 
+            elif node.op == "Rank":
```

Comment:
@lixiaoquan 

implementation look good.

But, I suggest to keep the them outside the ```GraphProto``` like a normal operator.

`params` is available there too, and the function can return the new Variable. 
```_num_params``` has no significance and can be ignored.

Reply:
I agree with you. I just forgot params can be changed there..